src/olympia/lib/settings_base.py

```diff
@@ -1279,16 +1279,16 @@ def get_db_config(environ_var, atomic_requests=True):
             'class': 'django_statsd.loggers.errors.StatsdHandler',
         },
     },
-    'root': {'handlers': ['mozlog'], 'level': logging.DEBUG},
+    'root': {'handlers': ['mozlog'], 'level': logging.INFO},
     'loggers': {
         'amo': {
             'handlers': ['mozlog'],
             'level': logging.DEBUG,
```

Comment:
are we keeping this on at `DEBUG` level on purpose?

Reply:
Good catch, this one should also be `INFO`.